pkg/factory/risingwave_object_factory.go

```diff
@@ -1665,6 +1665,15 @@ func (f *RisingWaveObjectFactory) buildPodTemplateFromNodeGroup(component string
 		})
 	}
 
+	// set RWC endpoint for frontend.
+	if component == consts.ComponentFrontend {
+		container := &podTemplate.Spec.Containers[0]
+		container.Env = append(container.Env, corev1.EnvVar{
+			Name:  envs.RWSbcAddr,
+			Value: "rpc://serverless-backfill-controller:1298",
```

Comment:
Apparently, we neither set values for non fundamental (necessary for bootstrap) envs in the operator, nor use hard-coded values for endpoints of other services. Fundamentally wrong.